.gitlab/ornl-ci.yml

```diff
@@ -0,0 +1,86 @@
+
+stages:
+  - setup
+  - update
+  - build ascent
+  - test
+  - clean
+
+Spack Setup:
+  stage: setup
+  tags:
+    - nobatch
+  variables:
+    SPACK_WORK_DIR: "/gpfs/wolf/proj-shared/csc449/ci/"
+    SPACK_ENV_NAME: "ascent"
+  script:
+    - if [ ! -d $SPACK_WORK_DIR/spack ]; then exit 0 fi
```

Comment:
`if [ -d $SPACK_WORK_DIR/spack ]; then exit 0; fi`? (if spack-dir exists, then this stage is skipped)